src/presentational/Classroom.jsx

```diff
@@ -40,20 +40,29 @@ export default class Classroom extends Component {
           return studentIds.indexOf( itm.id ) > -1;
         })
         .map((itm) => {
-          return itm.attributes.zooniverse_display_name
+          return itm. attributes
```

Comment:
There's an unexpected space between `itm.` and `attributes`, but this is a minor formatting hiccup. Code works fine.